Python/Github_Traffic/README.md

```diff
@@ -0,0 +1,85 @@
+# Github_Traffic
+
+This script, can help one determine all the unique viewers on his/her Repository.
+By using `access-token` one can determine the viewers and number of time they viewed the Repo.
+Currently data of only 14 days is possible (limitation of GitHib-API), but in future by using the GraphQL queries, this limitation can be answered.
+Te script uses PickleDB, the data can be imported in CSV as well, although the functionality hasn't been added till this point of time.
+
+## Note
+
+This script can be converted in a Bot, Action or GitHub app!!!
+
+
+## Dependency
+
+- githubpy
+- PickleDB
+
+These are summarised in `requirement.txt`
+
+
+## Setup
+
+1. A virtual environment (recommended)
+1. `pip install -r requirements.txt`
+1. Generate your own access token from [here](https://github.com/settings/tokens) (If you already have one with `repo` rights, it can be used as well)
+1. It is recommended to paste this token somewhere, as one cant review it again.
+1. Determine the Repository whose traffic you want to view.
+1. Run the Script
+
+## Usage
+
+Sample Usage -
+
+`
+python github_traffic.py collect -u vybhav72954 -r Music-Mood-Analysis -t *********
```

Comment:
Is it possible to have an `.env` file to store the Github Token. As a User, I don't like to copy past the Token everytime (Take it as optional for now)

Reply:
I will be honest i know this can lead to some security vulnerabilities, but am actually not sure how this worse, I will read about this for sure.